src/System.Management.Automation/commands/utility/FormatAndOutput/common/DisplayDatabase/displayDescriptionData.cs

```diff
@@ -713,6 +718,11 @@ public abstract class PSControl
         /// </summary>
         public bool OutOfBand { get; set; }
 
+        /// <summary>
+        /// Set explicit limit for Width. Default width is a console host width.
+        /// </summary>
+        public int? Width { get; set; }
+
```

Comment:
Oh, thinking about the property some more, I had some concerns. If we're thinking about files, width should really be `uint64`. To keep the code simpler, we could disallow values > `int.MaxValue`, but this would future proof the api to avoid adding `Width64`.